tests/functional/firefox/os/test_devices.py

```diff
@@ -0,0 +1,58 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from ....pages.firefox.os.devices import DevicesPage
+
+
+@pytest.mark.sanity
+@pytest.mark.nondestructive
+def test_family_navigation(base_url, selenium):
+    page = DevicesPage(base_url, selenium).open()
+    menu = page.open_family_navigation_menu()
+    assert menu.is_displayed
+
+
+@pytest.mark.sanity
+@pytest.mark.nondestructive
+def test_open_close_modal(base_url, selenium):
+    page = DevicesPage(base_url, selenium).open()
+    page.select_location('Germany')
+    modal = page.open_modal()
+    assert modal.is_displayed
+    modal.close()
+
```

Comment:
Hmm, we don't do this elsewhere else, as the modal is removed from the DOM once it closes? This is already asserted in the `close()` method as well.